DemoApp/DemoAppCoordinator.swift

```diff
@@ -114,7 +114,7 @@ final class DemoAppCoordinator: NSObject, UNUserNotificationCenterDelegate {
         // Channels with the current user
         let controller = ChatClient.shared
             .channelListController(query: .init(filter: .containMembers(userIds: [userCredentials.id])))
-        let chatList = DemoChannelListVC.make(with: controller)
+        let chatList = DemoChannelListVC.make(with: controller) as! DemoChannelListVC
```

Comment:
We should safely unwrap, I know it's the Demo App but still (shows best practice)

Reply:
Actually, this made me spot an issue. The `make` function should actually return the type based on `Self` so that when subclassing, we don't need to cast it 🙂 

Fixed ✅